test/unit.js

```diff
@@ -5,6 +5,9 @@ describe("ng-device-detector", function () {
 
     function loadDetector(userAgent, setup) {
         module("ng.deviceDetector", function (deviceDetectorProvider) {
+            deviceDetectorProvider.addCustom('MY_CUSTOM', {
```

Comment:
Custom test setup can be done via the `loadDetector` parameter: `setup`

Reply:
Ok, could you please briefly explain how to use loadDetector in that configuration please? I couldn't find a way to pass a custom user agent string to the test.